utils/src/main/java/software/amazon/awssdk/utils/StringUtils.java

```diff
@@ -107,7 +106,7 @@
  * <p>#ThreadSafe#</p>
  * @see java.lang.String
  */
-@ReviewBeforeRelease("Remove the methods we don't end up using (and we've removed software.amazon.awssdk.core.util.StringUtils).")
+//TODO Remove the methods we don't end up using (and we've removed software.amazon.awssdk.core.util.StringUtils).
```

Comment:
we can remove it? I don't see any unused methods

Reply:
Yeah, will do